pkgs/development/python-modules/axisregistry/default.nix

```diff
@@ -18,15 +19,18 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [
     fonttools
-    protobuf3
+    protobuf
   ];
   nativeBuildInputs = [
+    pythonRelaxDepsHook
     setuptools-scm
   ];
   nativeCheckInputs = [
     pytestCheckHook
   ];
 
+  pythonRelaxDeps = [ "protobuf" ];
```

Comment:
This can also be squashed to the `axisregistry: init` commit. Also, put the reason explained in the commit message in a comment, instead of in the commit message.